src/tests/end-to-end/common/page-controllers/page.ts

```diff
@@ -246,6 +260,10 @@ export class Page {
         });
     }
 
+    public async takePagesScreenshot(): Promise<void> {
```

Comment:
`takeScreenshot()` or `screenshot()` are probably fine here; this is within the `Page` class, so all of the methods are going to be assumed to be acting on the `Page` by default.